apps/escuela-frontend/studio/schemas/objects/resources/solution.js

```diff
@@ -3,34 +3,68 @@ import {MdAutoFixHigh} from 'react-icons/md'
 export default {
   name: 'solution',
   type: 'object',
-  title: 'Solution',
+  title: 'Solution to Exercise',
   icon: MdAutoFixHigh,
   fields: [
+    {
+      name: 'label',
+      title: 'Label',
+      type: 'string',
+      hidden: true,
+    },
     {
       name: 'title',
       title: 'Title',
       type: 'string',
+      validation: (Rule) => Rule.max(90),
     },
     {
       name: 'slug',
       title: 'Slug',
       type: 'slug',
       validation: (Rule) => Rule.required(),
       options: {
-        source: 'label',
+        source: 'title',
         maxLength: 96,
       },
     },
+    {
+      name: 'solutionType',
+      title: 'Solution Type',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+      options: {
+        list: [
+          {title: 'Solution to Exercise', value: 'exerciseSolution'},
+          {title: 'Solution to Extra Credit', value: 'extraCreditSolution'},
+        ],
+      },
+    },
```

Comment:
This adds a `solutionType`, either a solution to the exercise or a solution to the extra credit. 

<img width="643" alt="CleanShot 2022-11-09 at 12 45 25@2x" src="https://user-images.githubusercontent.com/57044804/200938158-def08e3c-4b36-43d5-8679-773ea967c07d.png">